src/frontend/rtos/rtos-common.ts

```diff
@@ -60,9 +77,9 @@ export abstract class RTOSBase {
     // Refresh the RTOS structures
     public abstract refresh(frameId: number): Promise<void>;
 
-    // Return a string that represents the RTOS state. Ideally, it should return a grid/table that is
-    // hosted in an upper level structure
-    public abstract getHTML(): string;
+    // Return a string tuple (html + style element content) that represents the RTOS state.
+    // Ideally, it should return a grid/table that is hosted in an upper level structure
+    public abstract getHTML(): [string, string];
```

Comment:
Perhaps an interface should be defined for return type like
```typescript
export interface HtmlInfo {
    html: string;
    css: string;
}
```

Reply:
Already talked to @mayjs about that and we left it as a tuple and wait for your feedback. As you also came up with this idea, i will change it 🙂